### DISCUSSION TITLE
Keyboard shortcut to copy the link of a Row within Outline

### DISCUSSION THREAD

**Discussion by aga5tya** [vincode-io/Zavala#236] [General]
I'm looking for a way to quickly copy the link to a row within the app to use that reference in other outlines. I can do this right now by clicking on Share and then copying the link for the row, but I wanted to check in if there is a quicker way to copy this via some keyboard shortcut. Thanks.

**Comment by aga5tya** [comment]
I see it actually only links to the outline itself and not specific rows; I am not sure if this is supported, but it would be very helpful for my use case; a lot of my outlines reference rows from other outlines.

**Comment by GaiusScotius** [comment]
I too would very much like to be able to link to specific blocks within an outline, and the more so if done in a manner compatible with Hookmark (https://hookproductivity.com). If you're no familiar, Hookmark is an app that bi-directionally links any two objects that can be identified by a URL (usually x-callback). Being able to link to the outline (file) level is useful, but being able to "deep link" to objects within a file more so.

**Comment by vincode-io** [comment]
I added #237 to track this feature request.

**Reply by jsismail** [reply]
Please consider supporting anchors to implement this feature, where it’s the anchor that’s linkable.

**Reply by jsismail** [reply]
Also, if there were a way to travers links forward and backward easily, that would be super helpful.

**Reply by vincode-io** [reply]
@jsismail We do provide backlinks. Is that what you are asking for?

**Reply by jsismail** [reply]
@vincode-io I'm not sure, to be honest.  What do you mean by "banklinks".  Can you point me at a menu option on Mac?

**Reply by GaiusScotius** [reply]
"Back links" are not something directly supported by macOS, but are implemented by many Markdown based note taking systems. When you insert a link in one note referencing another, the application automatically appends to the referenced note a link to the referring note -- hence the name "back link". The idea is that, when the Markdown is expanded to HTML and displayed, you can click on the link to navigate to an another page. At the foot of that page is a block showing links to all the pages which reference it.

Personally I don't like the concept: the application in which the initial link is created needs to be able to edit the targeted object to append the reverse (back) link so it only works with homogeneous file types; and it clutters notes with what can be extraneous footnotes detailing everything else that targets that note. I much prefer Hookmark as it abstracts the links into a separate application "above" the linked objects, which I think is much more powerful and certainly visually cleaner.
